agent/handlers/v2/response_test.go

```diff
@@ -207,6 +251,16 @@ func TestTaskResponseWithV4Metadata(t *testing.T) {
 		DockerConfig: apicontainer.DockerConfig{
 			HostConfig: aws.String(`{"LogConfig":{"Type":"awslogs","Config":{"awslogs-group":"myLogGroup"}}}`),
 		},
+		KnownPortBindingsUnsafe: []apicontainer.PortBinding{
+			{
```

Comment:
Question; is it worthwhile to add a test here test checks for the known port bindings to be empty list?